### PR TITLE
Add Logo

### PR DESCRIPTION
**Pull request by flynnduism**
Adds the Porter logo for the toc onboarding ([#533](https://github.com/cncf/toc/issues/533))

### CODE REVIEW

File: docs/themes/porter/static/src/sass/helm-home.scss [hunk 1]
```diff
@@ -56,7 +56,7 @@ nav.home-nav {
 
     img,
     svg {
-      min-height: 150px;
+      // min-height: 150px;
```
Comment (carolynvs) [inline docs/themes/porter/static/src/sass/helm-home.scss:59]: There is still a css rule that is making the logo stretched on the homepage

```css
nav.home-nav h1 img, nav.home-nav h1 svg {
    min-height: 150px;
}
```

File: docs/themes/porter/static/src/sass/helm-mixins.scss [hunk 2]
```diff
@@ -8,6 +8,16 @@
 
 
 // 1.
+// Porter Colours
+
+$gray:   #3C2E53; // ideal for text
```
Comment (carolynvs) [inline docs/themes/porter/static/src/sass/helm-mixins.scss:13]: The colors look great on the docs page and nav!

**Review by carolynvs — CHANGES_REQUESTED** [review]
Not sure if the assets need to be regenerated to pick up the min-height fix or if there's another rule somewhere?

**Review by vdice — COMMENTED** [review]
Looks great to me!  Also seeing the stretched homepage logo as mentioned.  Defer to Carolyn for full review.

Two notes, both of which perhaps are best tracked by follow-ups:

1. My Chrome tab still shows the old logo:

Screen Shot 2020-09-22 at 10 10 59 AM

2. Do we wish to keep the DeisLabs logo/etc. at the bottom of the homepage as well?

Screen Shot 2020-09-22 at 10 11 05 AM

**Comment by carolynvs** [comment]
The removal of the deislabs logo is part of #1253 that I am working on today.

File: docs/themes/porter/static/src/sass/helm-home.scss (hunk 1 above)
Comment (flynnduism) [reply docs/themes/porter/static/src/sass/helm-home.scss:59]: looking into this now

**Comment by flynnduism** [comment]
I’m still having getting the porter site to build locally - having reviewed the scss/css I think all this needs is for hugo to build and recompile the assets. 

Currently blocked though, something is missing or broke in my setup.

---

## Hugo / Make Errors:

Running `make docs-preview`, there is no error. The site does not render at `localhost`.

If I run `make docs` or `hugo --source docs/` i get:

```
hugo --source docs/
Total in 2 ms
Error: add site dependencies: create deps: create PathSpec: build filesystems: create main fs: symlinks not allowed in this filesystem

```

**Comment by carolynvs** [comment]
Hugo is very sensitive to version. This is why we switched to previewing inside a docker container. I am guessing that you may have a different version of hugo installed locally.

Right now hugo is failing because of an error during the scss build

```
$ make docs-preview
632d384f4d0a1b249b94c0d6f698804ac0b0bb0f39f93ff52093342d3f46436d

$ docker logs 632d384f4d0a1b249b94c0d6f698804ac0b0bb0f39f93ff52093342d3f46436d
Building sites … ERROR 2020/09/22 20:49:46 error: failed to transform resource: SCSS processing failed: file "/src/themes/porter/assets/sass/docs-sidebar.scss", line 194, col 17: Function darken is missing argument $amount.
ERROR 2020/09/22 20:49:47 error: failed to transform resource: SCSS processing failed: file "/src/themes/porter/assets/sass/docs-sidebar.scss", line 194, col 17: Function darken is missing argument $amount.
Error: Error building site: logged 2 error(s)
Total in 2157 ms
```

Since I am using the exact same version of huge that we require `0.53-ext`, I'm able to run `make docs` and see the same error as well.

**Comment by carolynvs** [comment]
In docs-sidebar.scss on line 194 is invalid

```css
.sidebar-buttons {
   ... 
   background: darken($green), 3.5%
   // used to be background: desaturate(lighten($blue1, 35%), 20%);
}

**Comment by flynnduism** [comment]
Thank you @carolynvs that sorted out my issues. The site compiles properly and the [preview looks better](https://5f6af92bd33cda00078363ca--porter.netlify.app/).

**Comment by carolynvs** [comment]
@vdice Browser's cache favicons and don't request them once they have them for the domain. You need to browse to https://deploy-preview-1294--porter.netlify.app/src/img/favicon.png and you'll see the updated cat logo (if you don't do a hard refresh), now go back to https://deploy-preview-1294--porter.netlify.app and then hard refresh. Now you should see the favicon appear correctly.

**Review by carolynvs — APPROVED** [review]
Everything looks adorbs on my end! 🐱 :bowtie: 💕

**Comment by carolynvs** [comment]
I found a few more images hiding out, doing another pass right now

**Pull request merged**